src/whylogs/io/local_dataset.py

```diff
@@ -0,0 +1,104 @@
+import os
+from typing import Any, Callable, List, Optional, Tuple
+import abc
+from PIL import Image
+
+EXTENSIONS = ('.csv', '.xls', '.jpg', '.jpeg', '.png', '.ppm', '.bmp',
+              '.pgm', '.tif', '.tiff', '.webp')
+
+
+def valid_file(fname):
+    extension = os.path.splitext(fname)[1]
+    return extension in EXTENSIONS
+
+
+def file_loader(path: str) -> Any:
```

Comment:
this is not just a file loader -it's an image file loader, right?

Reply:
file loader, right now just image, will extend to other files.